impl/maven-impl/src/main/java/org/apache/maven/impl/AbstractSession.java

```diff
@@ -134,7 +132,7 @@ public AbstractSession(
             List<RemoteRepository> repositories,
             List<org.eclipse.aether.repository.RemoteRepository> resolverRepositories,
             Lookup lookup) {
-        this.session = nonNull(session, "session");
```

Comment:
yes not ideal but its kind of clear it context of `NullPointerException` whats going on. I would keep it, remove all, or at least separate the concerns moving the "cannot be null" suffix into dedicated layer, avoiding DRY and feature envy here, as fluff and c&p.